verticapy/connection/utils.py

```diff
@@ -46,6 +46,16 @@ def get_connection_file() -> str:
     -------
```

Comment:
For the get_confparser, do we need an example? @oualib

Reply:
in that case, can you write it @mail4umar ?

I am writing the new example to not waste Tyler time.

Write it as comment directly.